generator/src/test/java/com/scottlogic/deg/generator/cucumber/ContainingRegex.feature

```diff
@@ -392,9 +474,21 @@ Scenario: Running a 'containingRegex' request alongside a lessThan constraint sh
       | "bb"  |
       | "abb" |
 
-Scenario: Running a 'containingRegex' request alongside a lessThanOrEqualTo constraint should be successful
-     Given there is a field foo
-       And foo is containing regex /[b]{2}/
+Scenario: containingRegex string run against a non contradicting not lessThan should be successful
+     Given foo is containing regex /[b]{2}/
+       And foo is anything but less than 3
```

Comment:
does this mean that the generated string should be of `length 3 or greater`?

if so, how does the output `"bb"` match?

Reply:
No, this is a lessThan constraint which only applies to numeric type data. For this test because the background specifies a type of string the lessThan 3 is ignored and only the regex matters.

Hence the result of `"bb"` is valid.

Reply:
ah, of course, thanks for the explanation.